scripts/test_pdf_header.py

```
@@ -1,8 +1,8 @@
 #!/usr/bin/env python3
 """Test script to check if we can replace python-magic with simple header checking."""
 
-import os
 import glob
```

Ruff (0.12.2)

`glob` imported but unused

Remove unused import: `glob`

(F401)